dev-python/lxml/lxml-4.8.0-r1.ebuild

```diff
@@ -34,12 +34,10 @@ BDEPEND="
 	virtual/pkgconfig
 	dev-python/cython[${PYTHON_USEDEP}]
 	doc? (
-		$(python_gen_any_dep '
-			dev-python/docutils[${PYTHON_USEDEP}]
-			dev-python/pygments[${PYTHON_USEDEP}]
-			dev-python/sphinx[${PYTHON_USEDEP}]
-			dev-python/sphinx_rtd_theme[${PYTHON_USEDEP}]
-		')
+		dev-python/docutils[${PYTHON_USEDEP}]
```

Comment:
Is this right?

I think we only need to build the docs once (and indeed can only install them once), so one implementation is OK.

Reply:
I think you were nearly right actually -- but keep python_gen_any_dep, like d_e_d sphinx does

Reply:
ignorance is apparently a powerful tool. ;)